manifests/daemon.pp

```diff
@@ -13,6 +13,7 @@
   $checksum_verify = false,
   $checksum      = '',
   $checksum_type = 'sha512',
+  $envs = {}
```

Comment:
why do you declare $envs when you never use it?

Reply:
I use it inside generic.env.erb